supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala

```diff
@@ -149,15 +149,19 @@ class JesAsyncBackendJobExecutionActor(override val standardParams: StandardAsyn
   }
 
   /**
-    * Turns WomFiles into relative paths.  These paths are relative to the working disk
+    * Turns WomFiles into relative paths.  These paths are relative to the working disk.
     *
     * relativeLocalizationPath("foo/bar.txt") -> "foo/bar.txt"
     * relativeLocalizationPath("gs://some/bucket/foo.txt") -> "some/bucket/foo.txt"
+    * relativeLocalizationPath("gs://some/bucket/foo.txt", keepPrefix = false) -> "foo.txt"
     */
-  private def relativeLocalizationPath(file: WomFile): WomFile = {
+  private def relativeLocalizationPath(file: WomFile, keepPrefix: Boolean): WomFile = {
```

Comment:
I don't see this called anywhere with `keepPrefix = false` ?

Reply:
Yeah, the parameter is a remnant from before I fixed `write_lines`. I'll remove